sdk/webpubsub/azure-messaging-webpubsub/README.md

```diff
@@ -125,6 +126,25 @@ When the client is connected, it can send messages to the upstream application,
 webPubSubServiceClient.sendToAll("Hello world!", WebPubSubContentType.TEXT_PLAIN);
 ```
 
+### Broadcast message to entire hub with filter
+
+```java readme-sample-broadcastToAll-filter
+// send a text message to the entire hub with a filter on userId
+BinaryData message = BinaryData.fromString("Hello World - Broadcast test!");
+webPubSubServiceClient.sendToAllWithResponse(
+    message,
+    WebPubSubContentType.TEXT_PLAIN,
+    message.getLength(),
```

Comment:
is this a must-have parameter? feels redundant, can we remove it in API surface?

Reply:
Yes, the content-type can also be binary, then we can't infer the length from azure-core, we need user to pass in length.

Reply:
To be more precise, the data could be from e.g. `FileInputStream`, in which case the `length` had to be provided by user.